### DISCUSSION TITLE
Windows 10 does not include sumatra on the list of  possible default pdf readers

### DISCUSSION THREAD

**Discussion by florinav** [sumatrapdfreader/sumatrapdf#2942] [Q&A]
Hi  Windows 10 does not include sumatra on the list of  possible default pdf readers

I can only use it individually, on each file, via open with. But open with does not have anymore the option Always open with. 

Any solution?

**Comment by GitHubRulesOK** [comment]
right click a pdf and see "opens with" has a change button 
ignore the icons and keep going down (more apps,  look for Another app) until you see the windows explorer find exe then go to where SumatraPdf.exe is and set it as default.

**Comment by kjk** [comment]
See https://www.sumatrapdfreader.org/docs/Set-as-default-PDF-viewer

**Reply by elisimpson** [reply]
"SumatraPDF registers itself for all supported file types (.pdf, .png etc.)."

This only appears to be true for the installed version. Is there a way to register it when using the portable version? Or alternately a list of all the filetypes Sumatra supports?

EDIT: I found this in the manual:
https://www.sumatrapdfreader.org/docs/Supported-document-formats

**Reply by GitHubRulesOK** [reply]
@elisimpson any file type can be registered as default to any app by right click e.g. a pdf and through the click options, however MS had a bad habit of clawing PDFs back to Edge Browser (and I think older/earlier Edge was ePub default but no more)

So it should stick in registry but do not be surprised if it needs reset after some Odd Tuesday updates.
